### ISSUE TITLE
working across timezones

### ISSUE THREAD

**Issue by dk4tz** [Alkaar/resy-booking-bot#90]
i got the sniper up, but it took the shot at the wrong time. 

@Alkaar can you please share how the bot understands time? is it GMT? ideally, it would be a parameter we pass to the resyConfig.

**Comment by dk4tz** [comment]
update: i saw the code uses date/time local to your machine... server i was running was on UTC. make sure you check the system time/zone by typing "date" into terminal before running. this will help you set the correct snipe time

**Comment by Alkaar** [comment]
Yes, you are correct. This is noted in the readme under the `How it works` section.
> The bot runs based on the local time of the machine it's running on

I think this is a good suggestion. I could have it as a parameter in the config that if it's set to empty, will default to the local time of the machine. Else it will use the time that you pass in the config. I'll leave this issue open as a possible feature enhancement.
